scripts/exgdas_enkf_sfc.sh

```diff
@@ -143,47 +138,65 @@ if [ $DOIAU = "YES" ]; then
             cmem=$(printf %03i $imem)
             memchar="mem$cmem"
 
-            [[ $TILE_NUM -eq 1 ]] && mkdir -p $COMOUT_ENS/$memchar/atmos/RESTART
+            MEMDIR=${memchar} YMD=${PDY} HH=${cyc} generate_com \
+                COM_ATMOS_RESTART_MEM:COM_ATMOS_RESTART_TMPL
+
+            MEMDIR=${memchar} RUN="enkfgdas" YMD=${gPDY} HH=${gcyc} generate_com \
+                COM_ATMOS_RESTART_MEM_PREV:COM_ATMOS_RESTART_TMPL
+
+            [[ ${TILE_NUM} -eq 1 ]] && mkdir -p "${COM_ATMOS_RESTART_MEM}"
 
-            $NCP $COMIN_GES_ENS/$memchar/atmos/RESTART/$bPDY.${bcyc}0000.sfc_data.tile${n}.nc $COMOUT_ENS/$memchar/atmos/RESTART/$bPDY.${bcyc}0000.sfcanl_data.tile${n}.nc
-            $NLN $COMIN_GES_ENS/$memchar/atmos/RESTART/$bPDY.${bcyc}0000.sfc_data.tile${n}.nc $DATA/fnbgsi.$cmem
-            $NLN $COMOUT_ENS/$memchar/atmos/RESTART/$bPDY.${bcyc}0000.sfcanl_data.tile${n}.nc $DATA/fnbgso.$cmem
-            $NLN $FIXfv3/$CASE/${CASE}_grid.tile${n}.nc                                $DATA/fngrid.$cmem
-            $NLN $FIXfv3/$CASE/${CASE}_oro_data.tile${n}.nc                            $DATA/fnorog.$cmem
+            ${NCP} "${COM_ATMOS_RESTART_MEM_PREV}/${bPDY}.${bcyc}0000.sfc_data.tile${n}.nc" \
+                "${COM_ATMOS_RESTART_MEM}/${bPDY}.${bcyc}0000.sfcanl_data.tile${n}.nc"
+            ${NLN} "${COM_ATMOS_RESTART_MEM_PREV}/${bPDY}.${bcyc}0000.sfc_data.tile${n}.nc" \
+                "${DATA}/fnbgsi.${cmem}"
+            ${NLN} "${COM_ATMOS_RESTART_MEM}/${bPDY}.${bcyc}0000.sfcanl_data.tile${n}.nc" \
+                "${DATA}/fnbgso.${cmem}"
+            ${NLN} "${FIXfv3}/${CASE}/${CASE}_grid.tile${n}.nc"     "${DATA}/fngrid.${cmem}"
+            ${NLN} "${FIXfv3}/${CASE}/${CASE}_oro_data.tile${n}.nc" "${DATA}/fnorog.${cmem}"
 
         done
 
-        $CYCLESH
+        CDATE="${PDY}${cyc}" ${CYCLESH}
```

Comment:
You have to set `CDATE` here because `CYCLESH` is from UFS-UTILS and uses `CDATE` still? If so, should we speak to George about converting `CDATE` to `PDYcyc` and leave a TODO here for now to change later?

Reply:
Yes